Dockerfile

```diff
@@ -0,0 +1,21 @@
+# default build is for CentOS7, change the base image to fit your build.
+FROM centos:centos7
+MAINTAINER Sebastien Le Digabel "sledigabel@gmail.com"
+
+#RUN yum update -y
+RUN yum install -y wget make rpmdevtools mock
+
```

Comment:
Dockerfile will be simpler and the build faster if you leave just the commands above in the Dockerfile (with some changes: remove commented out "yum update -y" and don't install mock), and the instructions would be:

    docker build -t kafka-build .
    docker run -ti -v $(pwd):/rpm -w /rpm kafka-build make